decidim-core/lib/decidim/form_builder.rb

```diff
@@ -251,20 +251,18 @@ def check_box(attribute, options = {}, checked_value = "1", unchecked_value = "0
     def date_field(attribute, options = {})
       value = object.send(attribute)
       data = { datepicker: "" }
-      data[:startdate] = I18n.localize(value, format: :datepicker) if value.present?
-      iso_value = value.present? ? value.strftime("%Y-%m-%d") : ""
+      data[:startdate] = I18n.localize(value, format: :datepicker) if value.present? && [Date, DateTime, Time].include?(value.class)
+      displayed_format = I18n.t("date.formats.displayed")
```

Comment:
I had another idea here. We could skip the new key altogether with a conversion method which would convert one format to the other. It doesn't look too hard, and we can iterate if new more complicated formats need to be supported in the future. For example, to support the "%d/%m/%Y" would be something like:

```ruby
def datepicker_date_format(ruby_format)
  ruby_format.gsub("%d", "dd").gsub("%m", "mm").gsub("%Y", "yyyy")
end
```

What do you think?

Reply:
I used `displayed` because first this key was used to translate the date format displayed to the user. But I agree that it now should be renamed.

Regarding the proposed new keys, why do we have a decidim_short format? why not configure Rails' `short` format instead? (which is the standard Rails way)
Using `datepicker` for the datepicker and user hinted format is great.

Reply:
I understand now why you used `displayed`, the format itself (not interpolated) is displayed in this case. Regarding your question about `decidim_short` vs `short`, I agree with you and I'm not sure why we're using our own format instead of customizing the Rails one... I guess we need some git archeology to find out :man_shrugging: